src/configure/helper/mustacheHelper.ts

```diff
@@ -3,11 +3,42 @@ import * as Mustache from 'mustache';
 export class MustacheHelper {
     public static getHelperMethods(): any {
         return {
-            "if": function () {
-                 /*
-                 * if returns first parameter if given clause is positive, otherwise second parameter
-                 * Usage: {{#if}}clause trueValue falseValue{{/if}}
+
+            "equals": function () {
+                /*
+                 * Usage: {{#equals}}value1 value2 true/false(ignorecase) returnIfTrue returnIfFalse(optional){{/regexReplace}}
```

Comment:
can you once verify the equals with Nitin and Tejas because I have seen different syntax in some templates..

Reply:
Verified with both of them